cli/azd/pkg/environment/environment.go

```diff
@@ -144,13 +139,38 @@ func (e *Environment) Getenv(key string) string {
 	return os.Getenv(key)
 }
 
+// Reloads environment variables from .env file
+func (e *Environment) Reload() error {
```

Comment:
Is this more appropriately named `LoadValues()`?

This currently does not load configuration, and only loads `env` variables.

Reply:
I almost included config reload as part of this but decided against it.  I'm open to either option.

Reply:
Updated to reload config as part of overall reload strategy.